tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java

```diff
@@ -404,6 +412,34 @@ public void visit(PropertyRemover propertyRemover) {
         ballerinaASTAPI.createStringLiteral(propertyRemover.getPropertyName());
         ballerinaASTAPI.endExprList(2);
         ballerinaASTAPI.createFunctionInvocation(true);
+    }
+
+    @Override
+    public void visit(VariableSetter variableSetter) {
+        createVariableOfTypeString(variableSetter.getValue(), variableSetter.getVariableName(), true, false);
+    }
+
+    @Override
+    public void visit(VariableRemover variableRemover) {
+        // createVariableOfTypeString("", variableRemover.getVariableName(), true,false);
+      /*  ballerinaASTAPI.createStringLiteral("");
+        ballerinaASTAPI.createVariable(variableRemover.getVariableName(), false);*/
```

Comment:
let's remove commented code blocks

Reply:
This was fixed with this commit >> https://github.com/wso2/product-ei/pull/879/files#diff-a7c35609bfca9af1252baf6d499e26feL534